wgpu/src/backend/webgpu.rs

```diff
@@ -1874,6 +1875,11 @@ impl crate::context::Context for ContextWebGpu {
         let module: &<ContextWebGpu as crate::Context>::ShaderModuleData =
             downcast_ref(desc.vertex.module.data.as_ref());
         let mut mapped_vertex_state = webgpu_sys::GpuVertexState::new(&module.0.module);
+        let _ = js_sys::Reflect::set(
+            &mapped_vertex_state,
+            &"constants".into(),
+            &hashmap_to_jsvalue(desc.vertex.compilation_options.constants),
+        );
```

Comment:
I don't like ignoring the error on this. If the WebGPU api still changes here (unlikely but who knows) or there's a bug here we'll never hear notice that the constants have been swallowed.
It should be possible to create a utility that encompasses all three callsites which internally does nothing if no constants are passed and otherwise uses js_sys::Reflect with error logging if something goes wrong.

I digged around a bit why `constants` is not exposed by wasmbindgen: It is mentioned here https://github.com/rustwasm/wasm-bindgen/blob/main/crates/web-sys/webidls/unstable/WebGPU.webidl#L635 but not exposed because of https://github.com/rustwasm/wasm-bindgen/issues/3587

A note linking to this issues should go into that utility as well (or each of these usage sites if utility method doesn't work out)

Reply:
That's totally fair - I have added a utility method as requested!